src/main/java/uk/ac/cam/cl/dtg/segue/api/AdminFacade.java

```diff
@@ -63,10 +63,14 @@
 import java.io.IOException;
 import java.security.InvalidKeyException;
 import java.security.NoSuchAlgorithmException;
+import java.util.ArrayList;
```

Comment:
This one is no longer needed.

Reply:
It did highlight it in my IDE at least, I just missed it 😬

Reply:
You can tell IntelliJ to automatically optimise the imports on save. I have not been able to make it work so that it also complies with the import rules set by CheckStyle, but Robert has, so he may be able to help you.